.bazelrc

```diff
@@ -14,6 +14,7 @@ build:asan --copt -fsanitize=address,undefined
 build:asan --linkopt -fsanitize=address,undefined
 build:asan --copt -fno-sanitize=vptr
 build:asan --linkopt -fno-sanitize=vptr
+build:asan --linkopt -fuse-ld=lld
```

Comment:
lld is already enabled in ci/build_setup.sh, though it is fine to have this here too.

Reply:
Yes, but when I build locally I also want `lld`.